### PR TITLE
Add Observability Operator page

### PR DESCRIPTION
**Pull request by slashpai**
Based on https://github.com/rhobs/observability-operator/pull/289#issuecomment-1562576311

cc: @simonpasquier

### CODE REVIEW

File: content/Products/ObservabilityOperator/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Observability Operator
+
+[observability-operator](https://github.com/rhobs/observability-operator) is a Kubernetes operator which enables the management of Monitoring/Alerting stacks through Kubernetes CRDs. Eventually it might also cover Logging and Tracing.
+
+The project relies heavily on the [controller-runtime](https://github.com/kubernetes-sigs/controller-runtime) library.
+
+> More details to follow in this document
+
+For now visit observability-operator [README](https://github.com/rhobs/observability-operator#readme) page for details.
```
Comment (simonpasquier) [inline content/Products/ObservabilityOperator/README.md:9]:
```suggestion
Check the observability-operator's [README](https://github.com/rhobs/observability-operator#readme) page for more details.
```

File: content/Products/Observability-Operator/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Observability-Operator
```
Comment (sthaha) [inline content/Products/Observability-Operator/README.md:1]:
```suggestion
# Observability Operator
```

File: content/Products/Observability-Operator/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+[observability-operator](https://github.com/rhobs/observability-operator) is a Kubernetes operator which enables the management of Monitoring/Alerting stacks through Kubernetes CRDs. Eventually it might also cover Logging and Tracing.
```
Comment (sthaha) [inline content/Products/Observability-Operator/README.md:3]:
```suggestion
[Observability Operator](https://github.com/rhobs/observability-operator) is a Kubernetes operator 
created in Go to setup and manage multiple highly available Monitoring Stack using Prometheus and
Thanos Querier.

 Eventually, this operator may also cover Logging and Tracing.
```

**Comment by slashpai** [comment]
@sthaha done

**Review by sthaha — APPROVED** [review]

**Pull request merged**